src/components/views/SuggestedViews/ViewCard.tsx

```diff
@@ -0,0 +1,45 @@
+import dayjs from 'dayjs';
+import { ReactEventHandler } from 'react';
+import { ZetkinView } from 'types/zetkin';
+import { Card, CardActionArea, Fade, Grid, Typography } from '@material-ui/core';
+import { makeStyles, Theme } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme: Theme) => ({
+    action: {
+        padding: theme.spacing(2),
+    },
+    content: {
+        minHeight: 200,
+    },
+}));
+
+type ViewCardProps = {
+    onClick: ReactEventHandler;
+    view: ZetkinView;
+}
+
+const ViewCard: React.FunctionComponent<ViewCardProps> = ({ onClick,  view }) => {
+    const classes = useStyles();
+
+    if (!view) return <div />;
```

Comment:
Personally I'm not a fan of single-line if statements. It obscures the complexity and this one actually caught me off-guard and I first thought there was an indentation mistake (and that the `return` below was conditional).

That said, we haven't forbidden them or even discussed it I believe. What do you think?

Reply:
Btw, why return `<div/>` and not `null`?

In fact, why can `view` ever be falsy when that's not reflected by the props type?

Reply:
@richardolsson we were just discussing this - getting excised now....